plugins/main/bridge/bridge.go

```diff
@@ -395,20 +405,45 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return fmt.Errorf("cannot set hairpin mode and promiscous mode at the same time.")
 	}
 
-	br, brInterface, err := setupBridge(n)
+	bridgeNS, err := ns.GetCurrentNS()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get current netns: %v", err)
+	}
+
+	defer bridgeNS.Close()
```

Comment:
duplicate close

Reply:
why is it duplicate? should close the CurrentNS retrieved before.

But make more sense to just move it into the else